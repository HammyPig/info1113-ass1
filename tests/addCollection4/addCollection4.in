add collection nofile
list all
add collection duplicates.csv
list all