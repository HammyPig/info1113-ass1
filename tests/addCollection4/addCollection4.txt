only add collections from files that exist