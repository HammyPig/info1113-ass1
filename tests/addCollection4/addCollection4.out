user: No such collection.

user: No books in system.

user: 3 books successfully added.

user: To Kill a Mockingbird (Harper Lee)
To Kill a Mockingbird (Harper Lee)
The Iliad (Homer)

user: Ending Library process.
